### ISSUE TITLE
3D Printer Nozzle load filament / unload filament

### ISSUE THREAD

**Issue by pedrolamas** [Templarian/MaterialDesign#7078]
### Before submitting a new icon request, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://pictogrammers.com/library/mdi) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](https://pictogrammers.com/docs/library/mdi/guides/brand-icons/) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I only intend to request a single icon (or a few near-identical ones) in this issue.

### What would you use this icon for?

I've been using the 3D Printer related icons for Fluidd (a Klipper web interface), and now require a couple more to identify the "Filament Load" and "Filament Unload" actions.

### Give us an example of what this icon might look like.

I would probably say that `printer-3d-nozzle` without the filament coming out of it (the bottom line) and then just add a line going IN the nozzle with an arrow down for the "Load Filament" and an arrow up for the "Unload Filament" actions.

**Comment by MrGrigri** [comment]
What about something like these? 

![printer-3d-nozzle-load](https://user-images.githubusercontent.com/4528551/232255151-263a3354-340f-44d9-b847-cd255ba0f4e2.png) ![printer-3d-nozzle-unload](https://user-images.githubusercontent.com/4528551/232255181-4267358b-8060-4b9f-b506-a987f27e14f4.png)

![printer-3d-nozzle-load-outline](https://user-images.githubusercontent.com/4528551/232255206-45215ee5-de55-43a4-87f6-4e4a023310c2.png) ![printer-3d-nozzle-unload-outline](https://user-images.githubusercontent.com/4528551/232255217-08a0c6b2-601f-41f0-993b-1004b5c5bbc1.png)

---

![printer-3d-nozzle-load](https://user-images.githubusercontent.com/4528551/232255249-0c889127-36e8-40b9-b250-b585eef832a7.svg) ![printer-3d-nozzle-unload](https://user-images.githubusercontent.com/4528551/232255251-cb9ed63f-a0a5-4912-9799-8350914e52fd.svg)

![printer-3d-nozzle-load-outline](https://user-images.githubusercontent.com/4528551/232255246-dc81a09f-0681-4a1d-894c-f4720c42f45c.svg) ![printer-3d-nozzle-unload-outline](https://user-images.githubusercontent.com/4528551/232255250-7370013d-c5d4-4f90-af83-85001285fc1e.svg)

**Comment by MrGrigri** [comment]
Looking at #7079, which one are you in need of? Both?

**Comment by pedrolamas** [comment]
Got to agree with you there @MrGrigri, those do look quite good and appropriate!

And yes, I will require both "load" (down) and "unload" (up) versions

**Comment by pedrolamas** [comment]
At the same time @MrGrigri, I do wonder if the ones you proposed would be more appropriate for "Extrude filament" and "Retract filament" actions, as we see the filament coming out of the nozzle...

Both are related, but the "Load/Unload filament" are actions to change the filament type, where as the "Extrude/Retract filament" are basically to make filament come out of the nozzle, or to make it go up the nozzle a bit (regularly done when the nozzle moves to a new printing position)

Bottom line for me is that there is a difference between moving the filament up/down the top of the nozzle (unload/load) and moving it up/down the bottom of the nozzle (retract/extrude)

**Comment by pedrolamas** [comment]
IMO, something like this might be more adequate for the intended "Load/Unload Filament" purpose!

![image](https://user-images.githubusercontent.com/85504/232307505-7771cf21-d8f9-46b4-a775-2256b3fa0176.png) ![image](https://user-images.githubusercontent.com/85504/232307526-86623cfb-71f3-4255-87bb-ada7d839f7a0.png)

![image](https://user-images.githubusercontent.com/85504/232307538-1348bb17-9f82-4fe5-ba10-1fcb7113f5a4.png) ![image](https://user-images.githubusercontent.com/85504/232307546-c146a53b-dec2-46ef-84aa-3de867800d13.png)

**Comment by meteyou** [comment]
@pedrolamas good idea for this icon request. I only think the arrow is very small in your your example for small icons.

**Comment by pedrolamas** [comment]
Good point @meteyou, I've updated my proposal by moving the arrow to the side and just leaving a line like filament on top of the nozzle
